manifests/e/EclipseAdoptium/Temurin/17/JDK/17.0.4.101/EclipseAdoptium.Temurin.17.JDK.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.17.JDK
+PackageVersion: 17.0.4.101
+MinimumOSVersion: 10.0.0.0
+InstallerSwitches:
+  Custom: INSTALLLEVEL=3
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.4.1%2B1/OpenJDK17U-jdk_x64_windows_hotspot_17.0.4.1_1.msi
+  InstallerSha256: CB0A70D663FA48729F9CFA80D655214E728406A1460F2FA744B614B98972A964
+  ProductCode: '{38B5ECF2-F859-46E3-9BA3-15C6E3C07942}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Eclipse Temurin JDK with Hotspot 17.0.4.1+1 (x64)
+- Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.4.1%2B1/OpenJDK17U-jdk_x86-32_windows_hotspot_17.0.4.1_1.msi
+  InstallerSha256: B104AD00DB3D83FFFBA8F8582E5A96614917449057E04D0A2B44C1515443F2CB
+  ProductCode: '{671AE74D-BCF3-4D25-A35B-F5625864947D}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Eclipse Temurin JDK with Hotspot 17.0.4.1+1 (x64)

```

```suggestion
  - DisplayName: Eclipse Temurin JDK with Hotspot 17.0.4.1+1 (x86)
```
This one is meant to be `(x86)` since `x86-32` is 32-bit.